resources/docker-images/README.md

```diff
@@ -0,0 +1,60 @@
+# Container
+
+Docker containers that can be used to build Keyman on the respective
+platforms. They contain everything that a CI build agent needs to
+build for the platform.
+
+## Prerequisites
+
+You'll need Docker Buildx installed to successfully be able to build the
+container images. This is most easily achieved by installing the [official
+Docker version](https://docs.docker.com/engine/install/ubuntu/).
+
+Currently it is not possible to use Podman instead of Docker due to a number
+of bugs and incompatibilities in the Podman implementation.
+
+## Building the images
+
+To build the docker images:
+
+```shell
+resources/docker-images/build.sh
+```
+
+By default this will create 64-bit images for building
+Keyman Core, Keyman for Android, Keyman for Linux and
+Keyman for Web. These images are based on the Ubuntu 24.04
+with Node 20 and Emscripten 3.1.58 (for the exact versions,
```

Comment:
Some of the Dockerfiles below use Node 18?
```
ARG REQUIRED_NODE_VERSION=18
```

Reply:
Should be node 20 -- I think the req shifted since this PR started